src/app/tabs/tabbed-menu.js

```diff
@@ -1,52 +1,98 @@
 var yo = require('yo-yo')
+var csjs = require('csjs-inject')
+var remixLib = require('remix-lib')
+
 var helper = require('../../lib/helper')
+var styles = require('../ui/styles-guide/theme-chooser').chooser()
 
-var css = require('./styles/tabbed-menu-styles')
+var EventManager = remixLib.EventManager
 
-class TabbedMenu {
+module.exports = class TabbedMenu {
   constructor (api = {}, events = {}, opts = {}) {
-    var self = this
-    var tabView = document.createElement('ul')
-    this.tabView = tabView
-    this.events = events
-    this.tabs = {}
-    this.contents = {}
-
-    events.app.register('debuggingRequested', () => {
```

Comment:
I think tests fails because of that being removed now

Reply:
oh, i will check - not sure what happened here